lib/internal/test_runner/reporter/junit.js

```diff
@@ -0,0 +1,146 @@
+'use strict';
+const {
+  ArrayPrototypeFilter,
+  ArrayPrototypeMap,
+  ArrayPrototypeJoin,
+  ArrayPrototypePush,
```

Comment:
These primordials are included in the "do not use" portion of the primordials documentation.

Reply:
Also, do we really need primordials in test runner?

Reply:
Yes. We need primordials in the test runner. Probably more so than in other parts of the codebase.